eq-author/src/App/questionPage/Routing/RoutingPage/RoutingEditor/RuleEditor/BinaryExpressionEditor/MultipleChoiceAnswerOptionsSelector.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "styled-components";
+
+import ToggleChip from "components/buttons/ToggleChip";
+import { includes, get } from "lodash";
+
+import { PropTypes } from "prop-types";
+
+const MultipleChoiceAnswerOptions = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 1em 0;
+`;
+
+class MultipleChoiceAnswerOptionsSelector extends React.Component {
+  static propTypes = {
```

Comment:
Can we utilise `propTypes` from [`graphql-anywhere`](https://github.com/apollographql/apollo-client/tree/master/packages/graphql-anywhere)?

Reply:
Unfortunately not. The parent is defining the fragment to be used by the two possible Components which depends on the type of the left side. Defining them in line was better but not ideal.